lib/cisco_node_utils/cmd_ref/snmpnotification.yaml

```diff
@@ -0,0 +1,11 @@
+# snmpnotification
+---
+enable:
+  config_get: "sh run snmp all | include 'snmp-server enable traps'"
+  config_get_token: '/^(no )?snmp-server enable traps %s ?$/'
```

Comment:
Still using `%s` printf style wildcards instead of `<trap_name>` key-value wildcard